tests/backend/unit/test_config.py

```
@@ -0,0 +1,924 @@
+        Example:
+            settings = Settings()
+            settings_dict = settings.model_dump()
+            # settings_dict["security"]["secret_key"] is a SecretStr, not a raw string
+        
+        Generated by CodeRabbit
+        """
+        settings = Settings()
+        
+        # Standard dict export should not expose secrets
+        settings_dict = settings.model_dump()
+        assert isinstance(settings_dict["security"]["secret_key"], SecretStr)
```

Ruff (0.11.9)

Undefined name `SecretStr`

(F821)